fiftyone/core/collections.py

```
@@ -11538,40 +11549,241 @@ async def _async_aggregate(self, aggregations, maxTimeMS=None):
+        branch_map,
+        compiled_aggs,
+        maxTimeMS=None,
+    ):
+        """Runs a consolidated $facet pipeline in parallel across _id partitions.
+
+        Returns a merged facet result document.
+        """
+        boundaries = await foo.get_id_boundaries(collection, n_partitions)
+        all_bounds = [None] + boundaries + [None]
+
+        kwargs = dict(allowDiskUse=True)
```

Ruff (0.15.9)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)